independent-publisher-connectors/InstagramBasicDisplay/apiDefinition.swagger.json

```diff
@@ -266,6 +266,58 @@
           }
         ]
       }
+    },
+    "/refresh_access_token": {
+        "get": {
+            "responses": {
+                "default": {
+                    "description": "default",
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "access_token": {
+                                "type": "string",
+                                "description": "A long-lived Instagram User Access Token.",
+                                "title": "Access Token"
+                            },
+                            "token_type": {
+                                "type": "string",
+                                "description": "bearer",
+                                "title": "Token Type"
+                            },
+                            "expires_in": {
+                                "type": "integer",
+                                "description": "The number of seconds until the long-lived token expires.",
+                                "title": "Expires In",
+                                "format": "int32"
+                            }
+                        }
+                    }
+                }
+            },
+            "summary": "Refreshes unexpired long-lived Instagram User Access tokens.",

```

Comment:
Summary should not end with period (.).  Please remove the period.

Reply:
The summary does not end with . anymore and I can't find any other similar errors.
<img width="578" alt="image" src="https://github.com/user-attachments/assets/96159c87-42f5-4a4a-b0a7-c2d254b90690">